std/path.d

```diff
@@ -2747,7 +2747,7 @@ else version (Posix)
     See_Also:
         $(LREF asAbsolutePath) which does not allocate
 */
-string absolutePath(return scope string path, lazy string base = getcwd())
```

Comment:
ditto

Reply:
`absoultePath` calls `chainPath(baseVar, path).array`, and `scope` inference fails for `.array` currently